tasks/gabrielsebrao/PJA-15/temperatureConverter.kt

```diff
@@ -0,0 +1,15 @@
+fun main() {
+    printFinalTemperature(27.0, "Celsius", "Fahrenheit" , { 9.0/5.0 * 27.0 + 32.0 })
+    printFinalTemperature(350.0, "Kelvin", "Celsius" , { 350.0 - 273.15 })
+    printFinalTemperature(27.0, "Celsius", "Fahrenheit" , {  5.0/9.0 * (27.0 - 32.0) + 273.15 })
+}
```

Comment:
this is not correct, you're converting celsius to Fahrenheit twice and with different values.

Reply:
okay, it's correct now, thank you. 

`printFinalTemperature(10.0, "Fahrenheit", "Kelvin" , { 5.0/9.0 * (10.0 - 32.0) + 273.15 })`

Reply:
okay, it's correct now, thank you.

     printFinalTemperature(10.0, "Fahrenheit", "Kelvin" , { 5.0/9.0 * (10.0 - 32.0) + 273.15 })